ci/kind/kind-setup.sh

```diff
@@ -133,7 +133,7 @@ function configure_networks {
   # remove old networks
   nodes="$(kind get nodes --name $CLUSTER_NAME | grep worker)"
   nodes=$(echo $nodes)
-  networks+=" kind"
+  networks+="$CLUSTER_NAME"
```

Comment:
I still think removing the space is incorrect here. Won't `for n in $networks; do` break?

If we are going from `network1 network2` to `network1network2`, the loop will consider a single invalid network name.